release-please-config.json

```diff
@@ -0,0 +1,136 @@
+    "src/apis/digitalassetlinks": {},
+    "src/apis/discovery": {},
+    "src/apis/displayvideo": {},
+    "src/apis/docs": {},
+    "src/apis/domainsrdap": {},
+    "src/apis/doubleclickbidmanager": {},
+    "src/apis/doubleclicksearch": {},
+    "src/apis/drive": {},
+    "src/apis/driveactivity": {},
+    "src/apis/eventarc": {},
+    "src/apis/factchecktools": {},
+    "src/apis/fcm": {},
+    "src/apis/file": {},
+    "src/apis/firebase": {},
+    "src/apis/firebasedatabase": {},
+    "src/apis/firebasedynamiclinks": {},
+    "src/apis/firebasehosting": {},
+    "src/apis/firebaseml": {},
+    "src/apis/firebaserules": {},
+    "src/apis/fitness": {},
+    "src/apis/games": {},
+    "src/apis/gamesConfiguration": {},
+    "src/apis/gamesManagement": {},
+    "src/apis/genomics": {},
+    "src/apis/gmail": {},
+    "src/apis/gmailpostmastertools": {},
+    "src/apis/groupsmigration": {},
+    "src/apis/groupssettings": {},
+    "src/apis/healthcare": {},
+    "src/apis/homegraph": {},
+    "src/apis/iam": {},
+    "src/apis/iap": {},
+    "src/apis/identitytoolkit": {},
+    "src/apis/indexing": {},
+    "src/apis/kgsearch": {},
+    "src/apis/libraryagent": {},
+    "src/apis/licensing": {},
+    "src/apis/lifesciences": {},
+    "src/apis/localservices": {},
+    "src/apis/manufacturers": {},
+    "src/apis/ml": {},
+    "src/apis/networkmanagement": {},
+    "src/apis/oauth2": {},
+    "src/apis/pagespeedonline": {},
+    "src/apis/people": {},
+    "src/apis/playablelocations": {},
+    "src/apis/playcustomapp": {},
+    "src/apis/plus": {},
+    "src/apis/poly": {},
+    "src/apis/prod_tt_sasportal": {},
+    "src/apis/pubsublite": {},
+    "src/apis/realtimebidding": {},
+    "src/apis/recommendationengine": {},
+    "src/apis/remotebuildexecution": {},
+    "src/apis/reseller": {},
+    "src/apis/run": {},
+    "src/apis/safebrowsing": {},
+    "src/apis/sasportal": {},
+    "src/apis/script": {},
+    "src/apis/searchconsole": {},
+    "src/apis/serviceconsumermanagement": {},
+    "src/apis/servicecontrol": {},
+    "src/apis/servicemanagement": {},
+    "src/apis/servicenetworking": {},
+    "src/apis/serviceusage": {},
+    "src/apis/sheets": {},
+    "src/apis/siteVerification": {},
+    "src/apis/slides": {},
+    "src/apis/smartdevicemanagement": {},
+    "src/apis/sourcerepo": {},
+    "src/apis/sql": {},
+    "src/apis/sqladmin": {},
+    "src/apis/storage": {},
+    "src/apis/storagetransfer": {},
+    "src/apis/streetviewpublish": {},
+    "src/apis/sts": {},
+    "src/apis/tagmanager": {},
+    "src/apis/testing": {},
+    "src/apis/toolresults": {},
+    "src/apis/tpu": {},
+    "src/apis/trafficdirector": {},
+    "src/apis/vault": {},
+    "src/apis/vectortile": {},
+    "src/apis/verifiedaccess": {},
+    "src/apis/webfonts": {},
+    "src/apis/webmasters": {},
+    "src/apis/workflowexecutions": {},
+    "src/apis/youtube": {},
+    "src/apis/youtubeAnalytics": {},
+    "src/apis/youtubereporting": {}
```

Comment:
Same questions as above on automation.  Is this generated?  Hand edited?

Reply:
I assume it will be automated eventually, I'd like to get us to an MVP where we have the existing libraries released.